sql/core/src/test/scala/org/apache/spark/sql/DataFrameNaFunctionsSuite.scala

```diff
@@ -36,6 +36,15 @@ class DataFrameNaFunctionsSuite extends QueryTest with SharedSQLContext {
       ).toDF("name", "age", "height")
   }
 
+  def createBooleanDF(): DataFrame = {
```

Comment:
Yup, right. I added it on top to keep both together, but it's only used for the boolean tests